mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -413,6 +463,8 @@ CLASS aij net/minecraft/entity/Entity
 	METHOD d getOppositeRotationVector (FF)Lcry;
 		ARG 1 pitch
 		ARG 2 yaw
+	METHOD d calculatePoseBounds (Lajc;)Lcrt;
```

Comment:
I'd suggest `calculateBoundsForPose`. It would make it slightly clearer that it is calculating the bounds of *the entity* for the given pose.

Reply:
I'm not a fan for using so many words in a method name like that. It gets into the territory of calling methods `calculateNewBoundsUsingTheInverseConstantLawOrThermalDynamics()`

I don't have any other objections to the name, so I'll go ahead and use that one.